src/Avalonia.Visuals/Media/PathFigure.cs

```diff
@@ -33,6 +51,31 @@ public PathFigure()
             Segments = new PathSegments();
```

Comment:
@jmacato I know this was a while ago, but do you remember why you marked `Segments` as nullable when it's always initialized with a non-null value in the constructor?

Reply:
i think it was to suppress a null warning, but yeah it doesnt make sense if it's already initialized

Reply:
Actually looking at WPF, it seems that even though `Segments` is initially non-null, it can be set to null so I think it's correct.